src/main/java/com/tekenable/config/prime/trial/cancer/glaxo/GlaxoCancerPrimer.java

```diff
@@ -106,7 +106,7 @@ protected void createTrial() {
             add(trialInfo);
         }});
 
-        final TrialSite trialSite1 = new TrialSite(trialInfo,
+        final TrialSite trialSite1 = new TrialSite(trialInfo, 1,
```

Comment:
first enough, it's on the way then. 
I meant that firt 2 params of new TrialSite constructor are actually not TrialSite specifc fields as all the rest of them (addresses and maps).